### ISSUE TITLE
Fatal error when no workflow is selected

### ISSUE THREAD

**Issue by haringsrob** [openeuropa/oe_editorial#32]
When you have a piece of content, and the content type is not having a workflow and you try to edit that workflow an error occurs.

```
Error: Call to a member function getTypePlugin() on null in
                                        Drupal\content_moderation\ModerationInformation->isDefaultRevisionPublished() (line 210 of
                                        /Users/rob/Sites/ec/eci-portal/web/core/modules/content_moderation/src/ModerationInformation.php) #0
                                        /Users/rob/Sites/ec/eci-portal/web/modules/contrib/oe_editorial/modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflo
                                        w.module(90): Drupal\content_moderation\ModerationInformation->isDefaultRevisionPublished(Object(Drupal\node\Entity\Node))
                                        #1
                                        /Users/rob/Sites/ec/eci-portal/web/modules/contrib/oe_editorial/modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflo
                                        w.module(64): _oe_editorial_corporate_workflow_is_last_revision_published(Object(Drupal\node\Entity\Node))
                                        #2 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/Extension/ModuleHandler.php(539):
                                        oe_editorial_corporate_workflow_form_node_form_alter(Array, Object(Drupal\Core\Form\FormState), 'node_oe_news_ed...')
                                        #3 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/Form/FormBuilder.php(835):
                                        Drupal\Core\Extension\ModuleHandler->alter('form', Array, Object(Drupal\Core\Form\FormState), 'node_oe_news_ed...')
                                        #4 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/Form/FormBuilder.php(277):
                                        Drupal\Core\Form\FormBuilder->prepareForm('node_oe_news_ed...', Array, Object(Drupal\Core\Form\FormState))
                                        #5 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/Controller/FormController.php(93):
                                        Drupal\Core\Form\FormBuilder->buildForm('node_oe_news_ed...', Object(Drupal\Core\Form\FormState))
                                        #6 [internal function]: Drupal\Core\Controller\FormController->getContentResult(Object(Symfony\Component\HttpFoundation\Request),
                                        Object(Drupal\Core\Routing\RouteMatch))
                                        #7 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123):
                                        call_user_func_array(Array, Array)
                                        #8 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/Render/Renderer.php(582):
                                        Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
                                        #9 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124):
                                        Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure))
                                        #10 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97):
                                        Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array)
                                        #11 /Users/rob/Sites/ec/eci-portal/vendor/symfony/http-kernel/HttpKernel.php(151):
                                        Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
                                        #12 /Users/rob/Sites/ec/eci-portal/vendor/symfony/http-kernel/HttpKernel.php(68):
                                        Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
                                        #13 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/StackMiddleware/Session.php(57):
                                        Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #14 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(47):
                                        Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #15 /Users/rob/Sites/ec/eci-portal/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(106):
                                        Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #16 /Users/rob/Sites/ec/eci-portal/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(85):
                                        Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #17 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(47):
                                        Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #18 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(52):
                                        Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #19 /Users/rob/Sites/ec/eci-portal/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23):
                                        Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #20 /Users/rob/Sites/ec/eci-portal/web/core/lib/Drupal/Core/DrupalKernel.php(693):
                                        Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
                                        #21 /Users/rob/Sites/ec/eci-portal/web/index.php(19):
                                        Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
                                        #22 /Users/rob/.composer/vendor/laravel/valet/server.php(158): require('/Users/rob/Site...')
                                        #23 {main}.
```

**Comment by nagyad** [comment]
Thanks, Rob. We will look into it asap.

**Comment by nagyad** [comment]
This problem is solved in the latest release 1.0.0-beta4.
